sites/gc-contributors/includes/gc-contributors.html

```diff
@@ -0,0 +1,8 @@
+<section class="gc-contributors">
+	<h2>{% if page.language == "en" %}From:{% else %}De&nbsp;:{% endif %}</h2>
```

Comment:
Because it is now inside the page details section, this heading must be downgraded for an heading level 3 (`<h3>`)